ui/src/app/cohort-search/constant.ts

```diff
@@ -19,3 +19,13 @@ export const DOMAIN_TYPES = [
     // { name: 'Vitals',       type: 'vitals' },
     // { name: 'Temporal',     type: 'temporal' }
 ];
+
+export const PM_UNITS = {
+    HEIGHT: 'cm',
+    WEIGHT: 'kg' ,
+    BMI: '',
+    WC: 'cm',
+    HC: 'cm',
+    BP: '',
+    'HR-DETAIL': 'beats/min'
```

Comment:
Nit: this appears to follow a different format?

Reply:
The '-' caused an error, so I put HR-DETAIL in single quotes. Maybe do the same for all properties for consistency?

Reply:
Yeah, I think doing it consistently would be good.

Reply:
Done!